platoon_strategic_IHP/test/main_test.cpp

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
There is a separate file in the test directory named .DS_Store.  It should be removed from the repo.  If you regularly work on a Mac, it might be good to add this filename to the .gitignore file so it doesn't show up again.